src/ui/utils/environment.ts

```diff
@@ -98,6 +98,12 @@ export function hasLoadingParam() {
   return getURL().searchParams.get("loading") != null;
 }
 
+export function getFocusRegion() {
+  const url = getURL();
+  const focusRegionParam = url.searchParams.get("focusRegion");
+  return focusRegionParam ? (decodeBase64FromURL(focusRegionParam) as TimeStampedPointRange) : null;
```

Comment:
Should we `try`/`catch` this in case? (I know it's not _new_ code, just code that's been moved.)

Reply:
Sure. Are you worried about the call to `decodeBase64FromURL`?

Reply:
Hm, I was, but then I looked and _that_ function catches and returns `undefined` so...my bad :smile:

Reply:
:)